.github/workflows/tests.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
     - name: Package snapshot changes
       if: ${{ failure() }}
       run: |
-        git ls-files -m BlueprintUICommonControls/Tests/Sources/ReferenceImages | xargs tar -cvf snapshot_changes_${{ matrix.sdk }}.tar
+        git ls-files -mo BlueprintUICommonControls/Tests/Sources/ReferenceImages | xargs tar -cvf snapshot_changes_${{ matrix.sdk }}.tar
```

Comment:
The [`-o` flag](https://git-scm.com/docs/git-ls-files#Documentation/git-ls-files.txt--o) includes untracked files. I changed this in order to grab the new snapshot images off of CI instead of running all these sims locally.

Reply:
TIL!